elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -0,0 +1,121 @@
+@import "../../pfe-sass/pfe-sass";
+$LOCAL: modal;
+
+$pfe-modal--breakpoint--medium: 640px;
+
+:host {
+	--pfe-modal--MaxWidth: 70vw;
+	--pfe-modal--MinWidth: 50vw;
+	--pfe-modal--MaxWidth--mobile: 94vw;
+	--pfe-modal--MaxHeight: 90vh;
+	--pfe-modal--Padding: calc(#{pfe-var(container-padding)} * 2) 0 calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2);
+	--pfe-modal__overlay--BackgroundColor: #{pfe-color(overlay)};
+	--pfe-modal--CloseButtonSize: calc(#{pfe-var(ui--element--size)} - 4px);
+	display: block;
+	position: relative;
+}
+
+:host(:not([has_header])) {
+	--pfe-modal--Padding: calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 4) calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2);
+}
+
+.pfe-modal {
+	&__outer {
+		display: flex;
+		position: fixed;
+		height: 100%;
+		width: 100%;
+		top: 0;
```

Comment:
```suggestion
		top: calc( (100vh - var(--pfe-modal--MaxHeight)) / 2);  // also suggested to set some top/bottom spacers based on the maxHeight var
		bottom: calc( (100vh - var(--pfe-modal--MaxHeight)) / 2); 
```

Reply:
This line won't work but you can do this which I believe works well: `top: calc(50vh - var(--pfe-modal--MaxHeight));` or this: `top: calc(100vh - calc(var(--pfe-modal--MaxHeight) / 2));` however nested calcs don't work in IE11 so you need a manual fallback for it if you do this.

Reply:
^ Reason being calc in IE11 runs into issues with nested calculations

Reply:
@starryeyez024 @castastrophe I think that the original intent of this code was to vertically center the modal with some margins above and below. We achieved this with flexbox and set a max-height on the window itself. Marking this resolved if it's okay with everyone?